.github/dependabot.yml

```diff
@@ -13,3 +13,5 @@ updates:
       - "approval: review pending"
     reviewers:
       - "openssl/committers"
+    cooldown:
+      default-days: 7
```

Comment:
Does this mean it will check for updates, but not react when in cooldown period?